### ISSUE TITLE
Overpass Semibold given Overpass Light family name

### ISSUE THREAD

**Issue by rgmatthes** [RedHatBrand/Overpass#49]
Prevents correct installation.

**Comment by andyfitz** [comment]
?  can you add more detail here

**Comment by rgmatthes** [comment]
Sure. I installed the full Overpass family on a desktop using the OTF files and noticed Overpass Semibold only seemed available as an italic. Digging in, I found that's because Overpass Semibold Regular is mistakenly for in the Overpass Light family, not the Semibold family.

Here's a screenshot:
![image](https://cloud.githubusercontent.com/assets/6545324/24711779/8c0b3a86-19e6-11e7-8953-224bbda21d8b.png)

**Comment by DelveFonts** [comment]
Hi @rgmatthes, Thank you for bringing this to our attention. The family name being “Overpass Light” is not a mistake. It is grouped with the Light so it can act as the ‘Bold’ for the Light weight in applications that make use of style-linking. This is fairly standard practice. Perhaps the real question here would be: why is it only showing up as italic? We will look into this matter. Could you tell us what OS / App is this occurring in? Thanks!

**Comment by rgmatthes** [comment]
This is in Windows 10, Microsoft Word 2016.

Here's a better picture illustrating my problem (screenshot from FontForge free trial):
![image](https://cloud.githubusercontent.com/assets/6545324/24757436/e53a72de-1aa4-11e7-8746-27acf28f1f6e.png)

tl;dr: I think the circled values should match.

**Comment by DelveFonts** [comment]
Thanks @rgmatthes! Yes, they should match. Not sure how that happened but it will be fixed.

**Comment by davelab6** [comment]
I don't recommend style linking anything except Roman 400 to Roman 700 and
Italic 40 to Italic 800

@m4rc1e did you hotfix this for GF?

**Comment by m4rc1e** [comment]
@davelab6 We've used the ttf, this problem is for the otfs. I also ran the ttfs through our naming sanitiser.

![screen shot 2017-04-10 at 11 13 28](https://cloud.githubusercontent.com/assets/7525512/24857044/c26e4748-1dde-11e7-955c-381c7c8c75cc.png)
My font menu look good with the GF ttfs.

**Comment by DelveFonts** [comment]
“I don't recommend style linking anything except Roman 400 to Roman 700 and
Italic 40 to Italic 800”

@davelab6 Why is that?

**Comment by davelab6** [comment]
It's the current fontlab recommendation; in my experience users are
confused by missing styles :)

**Comment by rgmatthes** [comment]
One more very picky thing: consistent hyphens in the variant names. This is the Windows 10 preview window for both Regular and Bold:
![image](https://cloud.githubusercontent.com/assets/6545324/25241358/4ae959d0-25bb-11e7-8577-0de854130984.png)

No feelings on whether these should be hyphened or not, just that they should be consistent. The registry also gets goofed on this:
![image](https://cloud.githubusercontent.com/assets/6545324/25241418/87c83402-25bb-11e7-9871-fb2916c3e6aa.png)

Not a top priority, but if you're messing with the properties already...

**Comment by m4rc1e** [comment]
@rgmatthes feel free to start a new issue with this bug.
